src/test/enhetstester/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/VilkårsvurderingServiceTest.kt

```diff
@@ -1,5 +1,6 @@
 package no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering
 
+import org.hamcrest.CoreMatchers.`is` as Is
```

Comment:
Hva er dette?

Reply:
Endringen kommer av forskjellige former for linting